song-server/src/test/java/bio/overture/song/server/service/FileModificationServiceTest.java

```diff
@@ -133,55 +131,18 @@ public void testFileUpdateWithPublishedAnalysis() {
 
     // No Change
     val noChangeRequest = new FileUpdateRequest();
-    val noChangeResponse =
-        fileModificationService.securedFileWithAnalysisUpdate(
-            DEFAULT_STUDY_ID, DEFAULT_FILE_ID, noChangeRequest);
-    assertFalse(noChangeResponse.isUnpublishedAnalysis());
-    assertEquals(noChangeResponse.getFileUpdateType(), NO_UPDATE);
-    assertEquals(noChangeResponse.getOriginalAnalysisState(), PUBLISHED);
-    assertEquals(noChangeResponse.getOriginalFile(), originalFile);
-    assertEquals(
-        noChangeResponse.getMessage(),
-        "Original analysisState 'PUBLISHED' was not changed since the fileUpdateType was 'NO_UPDATE'");
-
-    // Metadata Update
-    val metadataUpdateRequest =
-        FileUpdateRequest.builder()
-            .info(
-                object()
-                    .with(
-                        randomGenerator.generateRandomUUIDAsString(),
-                        randomGenerator.generateRandomUUIDAsString())
-                    .end())
-            .build();
-    val originalFile2 =
-        fileConverter.convertToFileDTO(fileService.securedRead(DEFAULT_STUDY_ID, DEFAULT_FILE_ID));
-    val metadataUpdateResponse =
-        fileModificationService.securedFileWithAnalysisUpdate(
-            DEFAULT_STUDY_ID, DEFAULT_FILE_ID, metadataUpdateRequest);
-    assertFalse(metadataUpdateResponse.isUnpublishedAnalysis());
-    assertEquals(metadataUpdateResponse.getFileUpdateType(), METADATA_UPDATE);
-    assertEquals(metadataUpdateResponse.getOriginalAnalysisState(), PUBLISHED);
-    assertEquals(metadataUpdateResponse.getOriginalFile(), originalFile2);
-    assertEquals(
-        metadataUpdateResponse.getMessage(),
-        "Original analysisState 'PUBLISHED' was not changed since the fileUpdateType was 'METADATA_UPDATE'");
-
-    // Content Update
-    val contentUpdateRequest =
-        FileUpdateRequest.builder().fileSize(originalFile2.getFileSize() + 77771L).build();
-    val originalFile3 =
-        fileConverter.convertToFileDTO(fileService.securedRead(DEFAULT_STUDY_ID, DEFAULT_FILE_ID));
-    val contentUpdateResponse =
-        fileModificationService.securedFileWithAnalysisUpdate(
-            DEFAULT_STUDY_ID, DEFAULT_FILE_ID, contentUpdateRequest);
-    assertTrue(contentUpdateResponse.isUnpublishedAnalysis());
-    assertEquals(contentUpdateResponse.getFileUpdateType(), CONTENT_UPDATE);
-    assertEquals(contentUpdateResponse.getOriginalAnalysisState(), PUBLISHED);
-    assertEquals(contentUpdateResponse.getOriginalFile(), originalFile3);
+    Exception exception = null;
+    try {
+      fileModificationService.securedFileWithAnalysisUpdate(
+          DEFAULT_STUDY_ID, DEFAULT_FILE_ID, noChangeRequest);
+    } catch (ServerException ex) {
+      exception = ex;
+    }
+    assertNotNull(exception);
```

Comment:
put back the old test, as it correctly tests the different fileUpdateTypes. The only thing to really tweak are the response messages if they have changed in this PR. For the CONTENT Update section of this test, it should fail. this can be tested with 
```java
SongErrorAssertions.assertSongError(() -> 
     fileModificationService.securedFileWithAnalysisUpdate(DEFAULT_STUDY_ID, DEFAULT_FILE_ID,
     contentUpdateRequest), 
      ILLEGAL_FILE_UPDATE_REQUEST);
```

then you can update the analysisState to UNPUBLISHED and then assert the fileUpdateType is `CONTENT_UPDATE'